custom_components/tesla_custom/services.yaml

```diff
@@ -29,3 +29,18 @@ api:
       required: true
       selector:
         object:
+
+polling_interval:
+  description: Set polling interval for updating fresh data from an awake car
+  fields:
+    scan_interval:
+      description: Number of seconds between each poll.  See https://github.com/alandtse/tesla/POLLING_POLICY.md for possible impacts
```

Comment:
Let's fix the links so we can merge this in. I think this should be ready right?
https://github.com/alandtse/tesla/wiki/Polling-policy